aptos-move/block-executor/src/captured_reads.rs

```diff
@@ -661,6 +660,37 @@ impl<T: Transaction> CapturedReads<T> {
         Ok(true)
     }
 
+    pub(crate) fn get_read_summary(
+        &self,
+    ) -> HashSet<InputOutputKey<T::Key, T::Tag, T::Identifier>> {
+        let mut ret = HashSet::new();
+        for (key, read) in &self.data_reads {
```

Comment:
Is the total_supply read included here?

Reply:
not the modification of total supply (or all transactions will have conflicts). but explicit read for governance proposal etc, should be here.